openmdao/components/jax_implicit_comp.py

```diff
@@ -0,0 +1,36 @@
+"""
+An ImplicitComponent that uses JAX for derivatives.
+"""
+
+import sys
+from openmdao.core.implicitcomponent import ImplicitComponent
+from openmdao.utils.jax_utils import jax
+from openmdao.utils.om_warnings import issue_warning
+
+
+class JaxImplicitComponent(ImplicitComponent):
```

Comment:
There really isn't much in the jax imp and exp components, aside from option values. I thought there might be something to prevent the user from defining compute or compute_partials.  Do we really need the components?

Reply:
Most of the machinery for jax lives in ExplicitComponent and ImplicitComponent, because my original intention was to allow for an entire Group to be 'jaxxed', which required converting normal openmdao components to jax at runtime.  It's not clear at this point whether the Group functionality will be useable or not, so for now everything stays where it is, but if we decide in the future that the Group stuff isn't viable we can move all of that functionality into JaxExplicitComponent and JaxImplicitComponent.